src/pages/settings/Payments/AddPayPalMePage.js

```diff
@@ -100,25 +86,15 @@ class AddPayPalMePage extends React.Component {
                         onPress={this.setPayPalMeUsername}
                         pressOnEnter
                         style={[styles.mt3]}
-                        isDisabled={this.state.payPalMeUsername === this.props.payPalMeData.description}
-                        text={this.props.payPalMeData.description
-                            ? this.props.translate('addPayPalMePage.editPayPalAccount')
-                            : this.props.translate('addPayPalMePage.addPayPalAccount')}
+                        isDisabled={_.isEmpty(this.state.payPalMeUsername.trim())}
+                        text={this.props.translate('addPayPalMePage.addPayPalAccount')}
                     />
                 </FixedFooter>
             </ScreenWrapper>
         );
     }
 }
 
-AddPayPalMePage.propTypes = propTypes;
-AddPayPalMePage.defaultProps = defaultProps;
+AddPayPalMePage.propTypes = {...withLocalizePropTypes};
```

Comment:
We don't have any additional types to define, so this makes more sense.

Reply:
i.e There is nothing to define. `withLocalizePropTypes` is already defined and imported, Defining it again is redundant so it seemed fine to me. 

Also, I checked whether we had such a definition in existing components. I saw we defined it in the following way if there are no other additional props i.e in `TermsAndLicenses` and `UnreadActionIndicator`.

So I was okay with it.

@pecanoro Any thoughts though?

Reply:
I don't think there are guidelines for this exactly so it's fine either way. If any of you feel passionate about it, it would be good to bring it up on Slack.